packages/export/src/excel.js

```diff
@@ -0,0 +1,73 @@
+import _ from 'lodash/fp.js'
+import writeXlsxFile from 'write-excel-file/node'
+
+const convertToExcelCell = (value) => {
+  return {
+    type: String,
+    typeof: 'string',
```

Comment:
left over from something I was using for column width, but I abandoned coercing values because already had some odd instances ie: zip codes that were being coerced to strings and numbers because some have a - within them.

I will remove it

Reply:
Reminder that this still needs to be done.